sprokit/processes/adapters/embedded_pipeline_extension.h

```diff
@@ -156,9 +156,9 @@ class embedded_pipeline_extension_registrar
   : public plugin_registrar
 {
 public:
-  embedded_pipeline_extension_registrar( kwiver::vital::plugin_loader& vpl,
-                    const std::string& mod_name )
-    : plugin_registrar( vpl, mod_name )
+  embedded_pipeline_extension_registrar( kwiver::vital::plugin_loader& vpl_,
```

Comment:
I wonder where "vpl" is used? I'd probably go with "loader" instead, and "module_name" (as IIRC was suggested elsewhere).